charts/kubermatic/templates/kubermatic-controller-manager-dep.yaml

```diff
@@ -95,6 +95,11 @@ spec:
         {{- else }}
         - -openshift-addons-list={{ join "," .Values.kubermatic.controller.addons.openshift.defaultAddons }}
         {{- end }}
+        - -backup-snapshot-dir=/backup-snapshots
+        - -backup-s3-endpoint={{ .Values.kubermatic.controller.etcdBackups.s3Endpoint }}
+        - -backup-s3-bucket={{ .Values.kubermatic.controller.etcdBackups.s3Bucket }}
+        - -backup-s3-access-key={{ .Values.kubermatic.controller.etcdBackups.s3AccessKeyId }}
+        - -backup-s3-secret-access-key={{ .Values.kubermatic.controller.etcdBackups.s3SecretAccessKey }}
```

Comment:
We probably shouldn't pass configuration as args or envvars in this case, as it would require a redeploy to change them. Maybe a configmap and secret for the credentials. This way it will be possible for external containers to access them as well.

Reply:
Removed the options -- they're no longer used or supported.